tools/initMigrationToPP/initMigrationToPP.ts

```diff
@@ -0,0 +1,141 @@
+
+    // Load provider
+    logger.info('Load provider');
+    const currentProvider = getProviderAdjustingMultiplierGas(initMigrationToPPParams, ethers);
+
+    // Load deployer
+    logger.info('Load deployer');
+    const deployer = await getDeployerFromParameters(currentProvider, initMigrationToPPParams, ethers);
+    logger.info('Using with: ', deployer.address);
+
+    const { type, polygonRollupManagerAddress, rollupID, newRollupTypeID } = initMigrationToPPParams;
+
+    // Load Rollup manager
+    const PolgonRollupManagerFactory = await ethers.getContractFactory('PolygonRollupManager', deployer);
+    const rollupManagerContract = PolgonRollupManagerFactory.attach(
+        polygonRollupManagerAddress,
+    ) as PolygonRollupManager;
+
+    const outputJson = {} as any;
+
+    if (type === transactionTypes.TIMELOCK) {
+        logger.info('Creating timelock tx to add default vkey...');
+        const salt = initMigrationToPPParams.timelockSalt || ethers.ZeroHash;
+        const predecessor = initMigrationToPPParams.predecessor || ethers.ZeroHash;
+        const timelockContractFactory = await ethers.getContractFactory('PolygonZkEVMTimelock', deployer);
+        const operation = genOperation(
+            polygonRollupManagerAddress,
+            0, // value
+            PolgonRollupManagerFactory.interface.encodeFunctionData('initMigrationToPP', [rollupID, newRollupTypeID]),
+            predecessor, // predecessor
+            salt, // salt
+        );
+        // Schedule operation
+        const scheduleData = timelockContractFactory.interface.encodeFunctionData('schedule', [
+            operation.target,
+            operation.value,
+            operation.data,
+            operation.predecessor,
+            operation.salt,
+            initMigrationToPPParams.timelockDelay,
+        ]);
+        // Execute operation
+        const executeData = timelockContractFactory.interface.encodeFunctionData('execute', [
+            operation.target,
+            operation.value,
+            operation.data,
+            operation.predecessor,
+            operation.salt,
+        ]);
+        logger.info(`scheduleData: ${JSON.stringify(scheduleData, null, 2)}`);
+        logger.info(`executeData: ${JSON.stringify(executeData, null, 2)}`);
+        outputJson.scheduleData = scheduleData;
+        outputJson.executeData = executeData;
+        // Decode the scheduleData for better readability
+        outputJson.decodedScheduleData = await decodeScheduleData(scheduleData, PolgonRollupManagerFactory);
+    } else if (type === transactionTypes.MULTISIG) {
+        logger.info('Creating calldata to initMigrationToPP from multisig...');
+        const tx = PolgonRollupManagerFactory.interface.encodeFunctionData('initMigrationToPP', [
+            rollupID,
+            newRollupTypeID,
+        ]);
+        outputJson.aggLayerGatewayAddress = polygonRollupManagerAddress;
```

Comment:
```suggestion
        outputJson.polygonRollupManagerAddress = polygonRollupManagerAddress;
```
Or just append this value before (or after) the if... else because it should be added in all conditions